### DISCUSSION TITLE
Why is my "SpyderCHECKR" chart not detected?

### DISCUSSION THREAD

**Discussion by kulakovaanna** [colour-science/colour-checker-detection#51] [General]
### Question

Hello, I'm trying to detect checkers but module can't find it. I've tried to increase brightness, centering palette, correct its perspectively. What's wrong with my images? Here's an example of them
![IMG_9536_b](https://user-images.githubusercontent.com/71021025/163202820-c865d5ad-8afb-4b1d-bdae-74ecc139593d.png)
![a2](https://user-images.githubusercontent.com/71021025/163203050-6da7a8ac-1eed-4eb2-9fa2-56c36710d3ff.JPG)
.

**Comment by KelSolaar** [comment]
Hi @ftn21,

Sorry, I missed that one! We do not support the **Datacolor SpyderCheckr** currently. Out of curiosity, in the example notebook, it shown how to render the segmentation result, did you try to see what was being segmented?

Cheers,

Thomas

**Reply by mexicantexan** [reply]
I was able to correctly detect and adjust for the datacolor colorcard with version 0.1.2 by throwing in my own swatches... want me to share my crappy comment-ridden code for it?

**Reply by kulakovaanna** [reply]
@mexicantexan yes, it’d be great, thank you)

**Reply by nitinitinitinitin** [reply]
> I was able to correctly detect and adjust for the datacolor colorcard with version 0.1.2 by throwing in my own swatches... want me to share my crappy comment-ridden code for it?

Hey can you please share the code?

**Comment by nitinitinitinitin** [comment]
> ### Question
> Hello, I'm trying to detect checkers but module can't find it. I've tried to increase brightness, centering palette, correct its perspectively. What's wrong with my images? Here's an example of them ![IMG_9536_b](https://user-images.githubusercontent.com/71021025/163202820-c865d5ad-8afb-4b1d-bdae-74ecc139593d.png) ![a2](https://user-images.githubusercontent.com/71021025/163203050-6da7a8ac-1eed-4eb2-9fa2-56c36710d3ff.JPG) .

hey were you able to detect the Spyderchekr, I want to do it but it does not work, can you help me out??
